SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -88,6 +88,26 @@ void UpdateChangelistHistory(TUniquePtr<FRepState>& RepState)
 	SendingRepState->HistoryStart = SendingRepState->HistoryStart % MaxSendingChangeHistory;
 	SendingRepState->HistoryEnd = SendingRepState->HistoryStart + NewHistoryCount;
 }
+
+void ForceReplicateOnActorHierarchy(USpatialNetDriver* NetDriver, AActor* RootActor, const AActor* OriginalActor)
+{
+	if (RootActor->GetIsReplicated())
+	{
+		if (RootActor != OriginalActor)
+		{
+			if (USpatialActorChannel* Channel = NetDriver->GetActorChannelByEntityId(NetDriver->PackageMap->GetEntityIdFromObject(RootActor)))
+			{
+				Channel->ReplicateActor();
```

Comment:
One problem here is we lose how much data this actor is spending on bandwidth (not something we currently use, however it was teed up to be worked on soon (see channel saturation). This also might not work nicely with the replication graph - would you mind enabling it in the testgyms, and giving your test scenario a spin?

Reply:
On the bandwidth, is this metric still relevant when not acting as a server anymore ?

Reply:
Tested with TestGym's replication graph enabled. Did not see any issues.

Reply:
Any risks with having other hierarchy Actors already in the list of Actors being replicated this tick, this Actor could get replicated twice? any issue with this?

Reply:
No but it'll do a lot of unnecessary work. I think if this was reworked so instead of immediately replicating the actor, we signal to the NetDriver or ReplicationGraph that they should include this actor in this frame's replication list, we should solve both the redundant replication, as well as the bandwidth issue.

Reply:
Or use ForceNetUpdate

Reply:
Thing is, when migrating we lose Authority on the actor, which will early out of the ReplicateActor (this is also why this code does not result in infinite recursion).

Reply:
Discussed offline - will create a follow up ticket to track this work

Reply:
https://improbableio.atlassian.net/browse/UNR-3336